drivers/ovsdriver.go

```diff
@@ -20,25 +20,42 @@ import (
 	"errors"
 	"fmt"
 	"os"
+	"regexp"
+	"strconv"
 	"strings"
 
 	log "github.com/Sirupsen/logrus"
-	"github.com/contiv/ofnet"
-	"github.com/vishvananda/netlink"
-
 	"github.com/contiv/netplugin/core"
 	"github.com/contiv/netplugin/netmaster/mastercfg"
+	"github.com/contiv/ofnet"
+	"github.com/vishvananda/netlink"
 )
 
 type oper int
 
 const maxIntfRetry = 100
 
+// OvsDriverConfig defines the configuration required to initialize the
+// OvsDriver.
+type OvsDriverConfig struct {
+	Ovs struct {
+		DbIP   string
+		DbPort int
```

Comment:
Just for my knowledge what is DbIP and DbIP ? I dont see it being used in this diff.